srcpkgs/python3-qmk/template

```diff
@@ -0,0 +1,30 @@
+# Template file for 'python3-qmk'
+pkgname=python3-qmk
```

Comment:
Since this is a utility, and not necessary a python3 lib, the package name can be just `qmk`

Reply:
Understood, changed it to just qmk